### DISCUSSION TITLE
which dos version for gaming

### DISCUSSION THREAD

**Discussion by retropwned** [joncampbell123/dosbox-x#5167] [Q&A] [answered]
which dos version is best for gaming.. more specific, which version is the all around best for 'most' games.

**Comment by joncampbell123** [comment, marked as answer]
Anything from MS-DOS 5.0 to Windows 98 "pure DOS mode" (DOS as it runs before Windows starts), take your pick.

MS-DOS 3.30 is too old and Windows ME crippled their DOS commands and removed the ability to boot into pure DOS mode.
